packages/web-components/src/dropdown/README.md

```diff
@@ -0,0 +1,176 @@
+
+<br />
+<hr />
+<br />
+
+### **Suggested Template**
+
+<br />
+
+`selectTemplate` from FastFoundation
+
+<br />
+<hr />
+<br />
+
+## **Accessibility**
+
+[W3 Select Spec](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-select-element.html)
+
+<br />
+
+### **WAI-ARIA Roles, States, and Properties**
+
+<br/>
+
+**`<fluent-dropdown>`**
+
+| Attributes              | value      | Description                                                                                                                                                                          |
+| ----------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `aria-activedescendant` |            | Indicates the currently active or focused child element within a composite component.                                                                                                |
+| `aria-controls`         |            | Identifies the element(s) whose contents or presence are controlled by the current element, establishing a relationship between a controlling element and the controlled element(s). |
+| `aria-expanded`         |            | Indicates the current state of a collapsible element.                                                                                                                                |
+| `aria-haspopup`         | `listbox`  | Indicates that the current element has a popup context menu or submenu.                                                                                                              |
+| `current-value`         |            | Stores the current value of an input element. [RFC](https://github.com/microsoft/fast/issues/5119)                                                                                   |
+| `role`                  | `combobox` | Specifies the purpose or type of an element in the context of user interface components.                                                                                             |
+| `tabindex`              | `0`        | Determines the order in which an element receives focus when navigated using the keyboard.                                                                                           |
+
+<br />
+
+**`<div class="listbox".. />`**
+
+| Attributes | value     | Description                                                                              |
+| ---------- | --------- | ---------------------------------------------------------------------------------------- |
+| `role`     | `listbox` | Specifies the purpose or type of an element in the context of user interface components. |
+
+<br />
+<hr />
+<br />
+
+## **Preparation**
+
+<br />
+
+### **Fluent Web Component v3 v.s Fluent React 9**
+
+<br />
+
```

Comment:
I think this needs to get worked out - I'd be surprised if there were not deltas here.